docs/components/web_wallet.md

```diff
@@ -34,3 +34,363 @@ User agent web wallet supports various wallet operations and also [DIDComm](http
 - **DID Connect**: This operation can be used by a website(or agent) to perform [DID Exchange](https://github.com/hyperledger/aries-rfcs/blob/master/features/0023-did-exchange/README.md) with web wallet for all future DID Comm operations.  
 - **Presentation Exchange**: This operation can be used by a website (a Relying Party or verifier) to request submission of proof from wallet in align with requester's [proof requirements](https://identity.foundation/presentation-exchange/). This request can also be made in conjuction with DID Connect operation, so that wallet can request further information from requester (a Relying party or verifier) over DID comm channel.
 
+## Login to wallet
+After successful login, web wallet will load [polyfill library](https://github.com/digitalbazaar/credential-handler-polyfill) and a web credential handler will be installed which will serve all your credential handler request as per [Credential Handler API](https://w3c-ccg.github.io/credential-handler-api/) (aka CHAPI) standards.
+
+In addition to that, user agent web wallet will perform some onboarding operations during first login of an user to support various DID comm operations which includes,
```

Comment:
These onboarding operations will change once the TrustBloc login & recovery components are up and running.

cc @Baha-sk @aholovko @DRK3

Reply:
Yes